Base/Python/slicer/tests/test_slicer_util_getNodes.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import slicer
+import slicer.util
+
+class SlicerUnitTestTest(unittest.TestCase):
+    def setUp(self):
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume1")
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume2")
+
+    def test_getNodesNoArg(self):
```

Comment:
You could have only `test_getNodes(self):` and move all `self.assert` in the same function.